content/rules/59br37/proposed.md

```diff
@@ -279,7 +279,7 @@ This [text node][] is fully [visible][] at a [viewport size][] of 640 by 512.
 
 <a class="example-link" title="Passed Example 2" target="_blank" href="https://w3.org/WAI/content-assets/wcag-act-rules/testcases/59br37/47d2a65e7d1fcc2ac9457a6283e35e82d68aa3ce.html">Open in a new tab</a>
 
-This [text node][] is [horizontally clipped][] using `text-overflow: ellipsis` at a [viewport size][] of 640 by 512. A link to a full version of the poem is also provided.
+This [text node][] is [horizontally clipped][] using `text-overflow: ellipsis` at a [viewport size][] of 640 by 512. This fulfills expectation 1. A link to a full version of the poem is also provided. The link to the full version is not required by this rule, but is necessary to satisfy [success criterion 1.4.4 Resize text][sc144].
```

Comment:
`[success criterion 1.4.4 Resize text][sc144]` is not converted into a link, see [preview](https://deploy-preview-339--wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/59br37/proposed/#passed-example-2).